packages/block/src/block.ts

```diff
@@ -223,9 +223,15 @@ export class Block {
   validateTransactions(stringError: true): string[]
   validateTransactions(stringError = false) {
     const errors: string[] = []
-
+    const self = this
     this.transactions.forEach(function (tx, i) {
```

Comment:
i'm sure you know, but if you update the forEach to arrow syntax you can save yourself needing to do `self = this`

```suggestion
    this.transactions.forEach((tx, i) => {
```

Reply:
@ryanio ah, I personally had actually no idea. That's a super-valuable hint. 😄  (this workaround is really annoying and not very beautiful).